neighsyncd/neighsync.h

```diff
@@ -11,10 +11,10 @@
 
 /*
  * This is the timer value (in seconds) that the neighsyncd waits for restore_neighbors
- * service to finish, should be longer than the restore_neighbors timeout value (60)
+ * service to finish, should be longer than the restore_neighbors timeout value (120)
  * This should not happen, if happens, system is in a unknown state, we should exit.
  */
-#define RESTORE_NEIGH_WAIT_TIME_OUT 70
+#define RESTORE_NEIGH_WAIT_TIME_OUT 180
```

Comment:
>180 [](start = 36, length = 3)

Could you comment what is the range for valid value? This one is also magic to me.

Reply:
This is also Zhenggen's request. Essentially, this value only need to be bigger than the restore_neighbor.py timeout. But since we are reading the potential DB settings, so he wants this value to be something big, like 180 seconds.

Reply:
Updated comments.